apps/mosaico/src/Main.purs

```diff
@@ -197,6 +211,43 @@ frontpage { htmlTemplate } _ = do
             (encodeJson $ map articleStubToJson mostReadArticles)
   pure $ TextHtml html
 
+staticPage
+  :: Env
+  -> { params :: { pageName :: String }}
+  -> Aff (Response ResponseBody)
+staticPage env { params: { pageName } } = do
+  staticPageResponse <- fetchStaticPage pageName
+  mostReadArticles <- Lettera.getMostRead 0 10 "" HBL true
+  case staticPageResponse of
+    StaticPageNotFound ->
+      let maybeMostRead = if null mostReadArticles then Nothing else Just mostReadArticles
+      in notFound maybeMostRead { params: {path: List.fromFoldable ["sida", pageName]} }
+    p -> do
+      mosaico <- liftEffect MosaicoServer.app
+      let staticPageContent :: Either JSX String
+          staticPageContent =  
+            case p of 
+              StaticPageResponse page -> Right page.pageContent
+              StaticPageOtherError -> Left Error.somethingWentWrong
+              StaticPageNotFound -> Left mempty
+      let staticPageJsx = 
+            case staticPageContent of 
+              Right pageContent -> 
+                DOM.div { className: "mosaico--static-page"
+                        , dangerouslySetInnerHTML: { __html: pageContent } 
+                        }
+              Left jsx -> jsx
+      let mosaicoString =
+            DOM.renderToString
+            $ mosaico
+              { mainContent: StaticPageContent staticPageJsx
+              , mostReadArticles
+              }
+      html <- liftEffect do
```

Comment:
This block seems very linear to me. `html` and `mosaicoTemplate` variables could be omitted with some `>>=` application. But it's fine as is, just a comment about how I'd write it.